src/applications/edu-benefits/10203/helpers.jsx

```diff
@@ -0,0 +1,25 @@
+// 1995-STEM related
+const isEdithNourseRogersScholarship = form =>
+  form.isEdithNourseRogersScholarship;
+
+export const isChapter33 = form =>
+  form.benefit === 'chapter33' || form.benefit === 'fryScholarship';
+
+const isEligibleForEdithNourseRogersScholarship = form =>
+  (isChapter33(form) || form.benefit === undefined) &&
+  isEdithNourseRogersScholarship(form) &&
+  (form['view:exhaustionOfBenefits'] ||
+    form['view:exhaustionOfBenefitsAfterPursuingTeachingCert']) &&
+  (form.isEnrolledStem || form.isPursuingTeachingCert);
+
+export const displayStemEligibility = form =>
+  isEdithNourseRogersScholarship(form) &&
+  !isEligibleForEdithNourseRogersScholarship(form);
```

Comment:
Is there a reason why you didn't use functions like these that area being exported from the 1995 helpers?

Reply:
Because those helpers are going to be removed in a later PR. All Stem Scholarship functionality will eventually be moved to this new form.